libraries/Adapters/Microsoft.Bot.Builder.Adapters.Facebook/FacebookAdapterBotComponent.cs

```diff
@@ -19,7 +20,7 @@ public override void ConfigureServices(IServiceCollection services, IConfigurati
         {
             if (FacebookAdapter.HasConfiguration(configuration))
             {
-                services.AddSingleton<FacebookAdapter>();
+                services.AddSingleton<IBotFrameworkHttpAdapter, FacebookAdapter>(sp => new FacebookAdapter(configuration));
```

Comment:
maybe this needs a comment - not super necessary because this can be traced back to this commit and this issue

Reply:
Thanks, added them and linked to the longer term issue that led to this https://github.com/microsoft/botbuilder-dotnet/issues/5583